content/aips/CERTORA-CONTINUOUS-FORMAL-VERIFICATION.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Certora Continuous Formal Verification
+author: Shelly Grossman (@shellygr)
+shortDescription: A continuation of proposal 6308 to improve the security of smart contracts built on top of Aave using a combination of formal verification and manual code review. 
```

Comment:
Better to refer if needed to the proposal id on the governance of the previous engagement. Not sure what is the `6308`

Reply:
Changed to proposal 66 (https://app.aave.com/governance/proposal/66/)